test/specs/pingback/pingback.ts

```diff
@@ -14,7 +14,7 @@ describe( 'Pingback', function () {
 		assert.strictEqual( sqlResult.includes( 'WikibasePingback-1.' ), true );
 
 		const result = await browser.makeRequest(
-			process.env.PINGBACK_BEACON_SERVER
+			'http://mediawiki.svc'
```

Comment:
Any particular reason not to keep this in a variable?

Reply:
Good eye. Yes, this one is complicated. Previous the value for `PINGBACK_BEACON_SERVER` (would now be `PINGBACK_BEACON_URL`) was set globally in `default.env` (which is now `test-services.env`) but only used once in this test. Meanwhile the host which composes that URL is set literally in `suites/pingback/docker-compose.override.yml` as well as in its URL form in `suites/pingback/LocalSettings.php`, which it was before as well. The decision here is to take a step backward until this is sorted more and make it a literal string in all 3 places instead of only in 2 of those places. I think this leaves the "problem" more clear and gives us space to answer to it if/when it comes up in the future. All of the rest of our global URL variables are named according to core Wikibase Suites services and are different than `PINGBACK_BEACON_URL`. 

Hope that make sense. It's a good catch, but I think at this point this is the right way to leave it vs further sorting within this branch. It is in my opinion no more, and maybe slightly less, ideal than before this PR.